plugin/core/sessions.py

```diff
@@ -1128,6 +1129,7 @@ def __init__(self, manager: Manager, logger: Logger, workspace_folders: List[Wor
         self._status_messages = {}  # type: Dict[str, str]
         self.diagnostics_manager = DiagnosticsManager()
         self._semantic_tokens_map = get_semantic_tokens_map(config.semantic_tokens)
+        self.show_inlay_hints = userprefs().show_inlay_hints or True
```

Comment:
This is the same as writing `self.show_inlay_hints = True`. But it is probably not what you wanted.

Reply:
thanks